client/src/components/AddColumnDialog/AddColumnDialog.tsx

```diff
@@ -17,6 +17,14 @@ const AddColumnDialog = (): JSX.Element => {
     setSide(addToSide);
   };
 
+  const handleSubmit = (): void => {
+    const board = addColumn(side, columnName);
+
+    if (!board) return;
+
+    setOpen(false);
+  };
```

Comment:
Does this work as intended from last night?

Reply:
It should work. I will test it again.